### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by MarijaKnezevic**

### CODE REVIEW

File: sddi-base/Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN set -ex && \
   ls -lah /wheels
 
 # ckanext-grouphierarchy ######################################################
-ARG CKANEXT_SDDI_VERSION="1.0.1"
```
Comment (BWibo) [inline sddi-base/Dockerfile:27]: There are a lot of new features, we need a minor version step `1.1.0`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,24 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+### Fixed
+
+- Fixed inconsistent base image and build image versions
+
+### Security
+
+### Deprecated
+
+### Known issues
+
```
Comment (BWibo) [inline CHANGELOG.md:26]: Please update the changelog. Reference the PRs/issues in the upstream repos.

File: README.md [hunk 3]
```diff
@@ -150,7 +150,7 @@ are alway pinned to a stable release number or commit hash.
 |---|---|:---:|:---:|:---:|---|
 | [`scheming`](https://github.com/MarijaKnezevic/ckanext-scheming) | `5c30bba` | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | Configure and share CKAN dataset metadata forms. |
 | [`hierarchy`](https://github.com/ckan/ckanext-hierarchy) | `v1.2.0` | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | Allows to organize organizations and groups in a hierarchy tree (nested groups/orgs). |
-| [`grouphierarchysddi`](https://github.com/tum-gis/ckanext-grouphierarchy-sddi) |  `1.0.1` | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | Extends `hierarchy` with pre-defined groups and topics of the SDDI concept. |
+| [`grouphierarchysddi`](https://github.com/tum-gis/ckanext-grouphierarchy-sddi) |  `1.0.2` | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | Extends `hierarchy` with pre-defined groups and topics of the SDDI concept. |
```
Comment (BWibo) [inline README.md:153]: `1.1.0`, see below.

File: sddi-base/Dockerfile [hunk 4]
```diff
@@ -192,6 +192,11 @@ RUN set -ex && \
   ckan config-tool "${CKAN_INI}" "scheming.dataset_schemas = ckanext.scheming:ckan_dataset.yaml" && \
   ckan config-tool "${CKAN_INI}" "scheming.presets = ckanext.scheming:presets.json ckanext.repeating:presets.json ckanext.composite:presets.json" && \
   ckan config-tool "${CKAN_INI}" "scheming.dataset_fallback = false" && \
+
```
Comment (BWibo) [inline sddi-base/Dockerfile:195]: Is it necessary to ut these settings here. If these are the default values, this should be set upstream in the plugin. is it required to specify these config variables in `production.ini`?

**Review by BWibo — CHANGES_REQUESTED** [review]

File: sddi-base/Dockerfile [hunk 5]
```diff
@@ -24,7 +24,7 @@ RUN set -ex && \
   ls -lah /wheels
 
 # ckanext-grouphierarchy ######################################################
-ARG CKANEXT_SDDI_VERSION="1.0.1"
+ARG CKANEXT_SDDI_VERSION="bugfix-admin-config"
```
Comment (BWibo) [inline sddi-base/Dockerfile:27]: If it is ready, make a release in the Upstream repo and put the new version here.
There are a lot of new features, we need a minor version step 1.1.0.

File: sddi-base/Dockerfile [hunk 6]
```diff
@@ -191,7 +191,7 @@ RUN set -ex && \
   ckan config-tool "${CKAN_INI}" "ckanext.spatial.search_backend = solr-bbox" && \
   ckan config-tool "${CKAN_INI}" "scheming.dataset_schemas = ckanext.scheming:ckan_dataset.yaml" && \
   ckan config-tool "${CKAN_INI}" "scheming.presets = ckanext.scheming:presets.json ckanext.repeating:presets.json ckanext.composite:presets.json" && \
-  ckan config-tool "${CKAN_INI}" "scheming.dataset_fallback = false" && \
+  ckan config-tool "${CKAN_INI}" "scheming.dataset_fallback = false" && \
```
Comment (BWibo) [inline sddi-base/Dockerfile:194]: Please remove the space at the end of the line. Then we don't have a change here.

**Review by BWibo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (MarijaKnezevic) [reply CHANGELOG.md]: It is updated now.
If you have any comments, please let me know.

File: README.md (hunk 3 above)
Comment (MarijaKnezevic) [reply README.md]: The grouphierarchy is now as [v1.1.0](https://github.com/tum-gis/ckanext-grouphierarchy-sddi/releases/tag/1.1.)

File: sddi-base/Dockerfile (hunk 5 above)
Comment (MarijaKnezevic) [reply sddi-base/Dockerfile]: It is changed

File: sddi-base/Dockerfile (hunk 6 above)
Comment (MarijaKnezevic) [reply sddi-base/Dockerfile]: It is removed

**Pull request merged**